src/lib/support/logging/CHIPLogging.h

```diff
@@ -296,5 +296,42 @@ bool IsCategoryEnabled(uint8_t category);
 
 #endif // CHIP_CONFIG_ENABLE_FUNCT_ERROR_LOGGING
 
+/*  @def ChipLogFormatX64
```

Comment:
nit: please remove `@def ` line here and below - this seems to just restarte code.